docs/user-guide/cli-install-configure-zosmf.md

```diff
@@ -1,17 +1,25 @@
 # Configuring z/OSMF
 
-Complete the following IBM z/OSMF configuration tasks for your Zowe CLI implementation.
+For client-side components to communicate with the mainframe, z/OSMF requires configuration to make this happen.
 
 :::info Required role: systems programmer
 :::
 
+Complete the following IBM z/OSMF configuration tasks for the implementation of [Zowe CLI](../user-guide/user-roadmap-zowe-cli.md), [Zowe Explorer for Visual Studio Code](../getting-started/user-roadmap-zowe-explorer.md), or the [Zowe IntelliJ Plug-in](../user-guide/intellij-install.md).
+
+:::note
+
+If you are connecting to the mainframe with other methods than a z/OSMF profile, you do not need to configure z/OSMF. Other connection options might include using FTP, or your custom API.
```

Comment:
@adam-wolfe @JillieBeanSim -- is this note accurate/sufficient re: other mainframe connections besides z/osmf profile?

Reply:
I would suggest rearranging to: "If you are connecting to the mainframe with **methods other** than a z/OSMF profile, ..."

Other than that, it looks OK.